Azure/communication-ui-library-android
# [Feature][Chat] Tap and Hold context menu Copy message

## Purpose

* Adds context menu feature for chat messages with a tap-and-hold gesture
* Adds copy action for message context menu

## Does this introduce a breaking change?



## Pull Request Type
What kind of change does this Pull Request introduce?



## Pull Request Checklist




## How to Test


* Join a chat thread and tap-and-hold on a message
* Choose the Copy action from the context menu that appears


## What to Check
Verify that the following are valid
* Clipboard should contain the text content of the message

## Screenshots
![Screenshot_1668032067](https://user-images.githubusercontent.com/43901/200953871-7e01c649-f660-4e24-960f-d03823428420.png)
